xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/viewers/attachments.js

```diff
@@ -218,29 +218,46 @@ require(['jquery', 'xwiki-events-bridge'], function($) {
       }
     })
   });
+  $(document).on("xwiki:livetable:loadingComplete", function(event, data) {
```

Comment:
You should listen to ``xwiki:livetable:docAttachments:loadingComplete``. See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.js#L204 .